app/admin/contributions.rb

```diff
@@ -131,6 +134,7 @@
   controller do
     def approve
       resource.approve!(current_user.id)
+      NotificationMailer.notify_fill_contribution_description(resource).deliver_now
```

Comment:
also would be nice to do this async (deliver_later)

Reply:
Done